conf/messages

```diff
@@ -1,2 +1,91 @@
 service.name = card-payment-frontend
 service.text = This is your new service
+
+
+## Non page specific
+continue = Continue
+error.there-is-a-problem = There is a problem
+
+
+## page titles
```

Comment:
Sorry started adding these and forgot to stash it before committing, I can remove if needs be